src/main/java/HW14/Fruits/Box.java

```diff
@@ -0,0 +1,51 @@
+package HW14.Fruits;
+
+import java.util.Collection;
+import java.util.LinkedList;
+
+public class Box<T extends Fruit>{
+    LinkedList<Fruit> box = new LinkedList<>();
+
+    public void addToBoxOne(Fruit fruit) {
+        if(box.isEmpty()){
+            box.add(fruit);
+        }
+        if(fruit instanceof Apple && box.get(0) instanceof Apple){
+            box.add(fruit);
+        } else if(fruit instanceof Orange && box.get(0) instanceof Orange){
+            box.add(fruit);
+        }else {
+            return;
+        }
+    }
+
+    public void addToBoxMulti(Fruit fruit, int amount) {
```

Comment:
```suggestion
    public void addToBoxMulti(T fruit, int amount) {
```
навіщо цей метод? 
якщо комусь потрібно, то він сам викличе add() в циклі.